autonomy/cli/deploy.py

```diff
@@ -138,6 +138,12 @@ def deploy_group(
     default=False,
     help="Include an ACN node in the deployment setup.",
 )
+@click.option(
+    "--use-testnet",
+    is_flag=True,
+    default=False,
+    help="Use tendermint testnet setup.",
```

Comment:
What do you mean with "tendermint testnet"?

Reply:
In the deployment setup we use tendermint testnet setup to establish the chain initially generated using the `tendermint testnet` command. Since we have support for establishing the chain using the ACN now, there is not need to include the testnet setup in the deployment build

Reply:
Where is this documented in the docs?

My understanding is that the only way core now works is with ACN @Adamantios ?

Reply:
No, core works both ways via `share_tm_config_on_startup`. The flag's default is `False`, so I believe that the default here should be `True`, i.e., using testnet command.

Reply:
Why should the flag be True? We don't want to use it by default.

Is the flag propagated correctly?

Reply:
Because afaict from the PR's code, setting `--use-testnet` to `True` means that we do not use ACN. So the default should match the `share_tm_config_on_startup`'s default intent.